Q14/Codeville.rb

```diff
@@ -0,0 +1,18 @@
+def solution(weights, limit)
+    # p weights.sort!
+    boats = 0
+    check = 0
+    index = 0
+  
+    weights.each do |x|
+      check += weights[index] 
+      if check + weights[index] >= 200
```

Comment:
200 should be in as variable, so that limit can be set as anything

Reply:
I've changed it a variable